08-mapping.Rmd

```diff
@@ -419,10 +419,10 @@ tm_shape(world) +
 
 The preceding code chunk demonstrates key features of faceted maps created with **tmap**:
 
-- Shapes that do not have a facet variable are repeated (the countries in `world` in this case).
-- The `by` argument which varies depending on a variable (`year` in this case).
-- `nrow`/`ncol` setting specifying the number of rows and columns that facets should be arranged into.
-- The `free.coords`-parameter specifying if each map has its own bounding box.
+- shapes that do not have a facet variable are repeated (the countries in `world` in this case),
+- the `by` argument which varies depending on a variable (`year` in this case),
+- the `nrow`/`ncol` setting specifying the number of rows and columns that facets should be arranged into.
```

Comment:
There's an inconsistent full stop on this line.

Reply:
Oops, indeed. I had missed one.